core/lib/eth_signer/src/raw_ethereum_tx.rs

```diff
@@ -47,6 +48,10 @@ pub struct TransactionParameters {
     pub max_fee_per_gas: U256,
     /// miner bribe
     pub max_priority_fee_per_gas: U256,
+    /// Max fee per blob gas
+    pub max_fee_per_blob_gas: Option<U256>,
+    /// blob versioned hashes
+    pub blob_versioned_hashes: Option<Vec<H256>>,
```

Comment:
@koloz193 q: do we compute these values once at tx creation? If so they should live in DB i suppose.

Reply:
yes, the versioned hashes correspond to the blobs being used and the blob gas should be whatever was used for the txn